sdk/face/azure-ai-vision-face/CHANGELOG.md

```diff
@@ -2,14 +2,27 @@
 
 ## 1.0.0-beta.2 (Unreleased)
 
+- Added support for the Large Face List and Large Person Group:
+  - Added client `LargeFaceListClient` and `LargePersonGroupClient`.
```

Comment:
Should we consider calling this `FaceLargeListClient` as suggested in one of the earlier reviews?

Reply:
We'd like to keep them aligned with REST API.